pkgs/top-level/python-packages.nix

```diff
@@ -19106,7 +19106,7 @@ let
     disabled = isPy3k;
 
     src = pkgs.fetchgit {
-        url = "http://git.liw.fi/cgi-bin/cgit/cgit.cgi/cliapp";
+        url = "git://git.liw.fi/cliapp";
```

Comment:
We prefer to use `http` as git (ssh) might be blocked on some enterprise networks.